faasmcli/requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==21.9b0
+black==22.3.0
```

Comment:
Unrelated to this PR, but fixes the python formatting checks that had broken as a result of psf/black#2964.